### PR TITLE
MULE-13514: AGW CFX Facade Module in the list of privileged access artifacts

### PR DESCRIPTION
**Pull request by adcarabajal**

### CODE REVIEW

**Comment by pablokraan** [comment]
Create a MULE issue for this and explain why you need this privileged access

File: core/src/main/resources/META-INF/mule-module.properties [hunk 1]
```diff
@@ -17,7 +17,6 @@ artifact.export.classPackages=org.mule.runtime.core.api,\
                               org.mule.runtime.core.api.el,\
                               org.mule.runtime.core.api.el.datetime,\
                               org.mule.runtime.core.api.endpoint,\
-                              org.mule.runtime.core.api.event,\
```
Comment (pablokraan) [inline core/src/main/resources/META-INF/mule-module.properties]: Why are you removing this package?

**Comment by adcarabajal** [comment]
Issue for PR
https://www.mulesoft.org/jira/browse/MULE-13514

File: core/src/main/resources/META-INF/mule-module.properties (hunk 1 above)
Comment (adcarabajal) [reply core/src/main/resources/META-INF/mule-module.properties]: Missing during rebase. Restored

**Pull request merged**